### ISSUE TITLE
accordion events fire before animation has completed

### ISSUE THREAD

**Issue by r3wt** [zurb/foundation-sites#7709]
related issue #7708 

I would like to scroll the opened accordion item into view when it opens, but the event fires before the close animation is complete, resulting in the positioning code for the animated scroll being incorrect.
```js
$('.accordion').on('down.zf.accordion',function(event,target){
	event.preventDefault();
	$('html, body').animate({
		scrollTop: $(this).find('.accordion-item.is-active').eq(0).offset().top - 5,
	},300);
	return false;
});
```
this can be reproduced by opening an accordion item that is below the current opened item. you will see that the event clearly fires before the previous item has finished closing. In my opinion, this event should not fire until both close of previous panel and open of current panel animations did complete.

**Comment by natewiebe13** [comment]
Should be fixed in #7727

**Comment by zurbchris** [comment]
Indeed, fixed by #7727

**Comment by r3wt** [comment]
thanks guys. i appreciate your hard work in resolving this bug.
